src/utility_windows.go

```diff
@@ -242,3 +242,19 @@ func getLocalUser(userName string) (shared.LocalUser, error) {
 	}
 	return shared.LocalUser{}, nil
 }
+
+func getLocalServiceStatus(serviceName string) (shared.Service, error) {
+    serviceDataList, err := wapi.GetServices()
+    var serviceStatusData shared.Service
```

Comment:
you can remove this entirely and set the failed values to nil

Reply:
wdym

Reply:
see below